azure-batch/azure/batch/models/__init__.py

```diff
@@ -9,6 +9,10 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
+######DO NOT REMOVE######
```

Comment:
You actually don't need to do that, you should be able to inject this from the main `patch` in the client, using importlib to improt "models". I don't guarantee I'm adding this patch point in Autorest.